### PR TITLE
Gt 699 hapi removal

### PR DESCRIPTION
**Pull request by marianbuenosayres**
Removes the need for HAPI fhir to be part of the webapp classloader, allowing different services to use different versions of FHIR

### CODE REVIEW

**Review by boxwala — DISMISSED** [review]

File: a2d2-api/pom.xml [hunk 1]
```diff
@@ -150,7 +150,7 @@
 			<artifactId>encoder</artifactId>
 			<version>1.2.3</version>
 		</dependency>
-		<dependency>
+		<!--dependency>
```
Comment (hankwallace) [inline a2d2-api/pom.xml:153]: please remove these commented-out sections.

File: fhir-resource-wih/src/test/java/io/elimu/a2d2/fhir_resource_wih/ResourceWriteDelegateTest.java [hunk 2]
```diff
@@ -24,14 +26,16 @@
...
 import ca.uhn.fhir.context.FhirContext;
 import ca.uhn.fhir.rest.api.EncodingEnum;
 import ca.uhn.fhir.rest.client.api.IGenericClient;
 import ca.uhn.fhir.rest.client.impl.BaseClient;
 import io.elimu.a2d2.exception.FhirServerException;
+import io.elimu.a2d2.fhirresourcewih.FHIRDelegateHelper;
 import io.elimu.a2d2.fhirresourcewih.ResourceWriteDelegate;
 
-@Ignore ("disable for now as the test is based on public testing server")
+//@Ignore ("disable for now as the test is based on public testing server")
```
Comment (hankwallace) [inline fhir-resource-wih/src/test/java/io/elimu/a2d2/fhir_resource_wih/ResourceWriteDelegateTest.java:38]: please remove this commented-out line

**Review by hankwallace — COMMENTED** [review]
A few minor comments, then it's good to go.

**Review by hankwallace — APPROVED** [review]
Good to go

**Pull request merged**